storage/tianmu/core/tianmu_attr_exqp.cpp

```diff
@@ -898,8 +908,8 @@ void TianmuAttr::EvaluatePack_BetweenInt(MIUpdatingIterator &mit, int dim, Descr
       if (d.op == common::Operator::O_BETWEEN && !mit.NullsPossibleInPack(dim) && dpn.numOfNulls == 0) {
         // easy and fast case - no "if"s
         for (uint32_t n = 0; n < dpn.numOfRecords; n++) {
-          auto v = p->GetValInt(n);
-          if (pv1 > v || v > pv2)
+          uint64_t v = p->GetValInt(n);
```

Comment:
`GetValInt` will return a signed int value.

Reply:
Here it should be cast to `uint64_t`, as all the value stored in [0, uint64_max].